### DISCUSSION TITLE
I'd like to install stock e2s firmware on a e2 and use it like a stock e2s

### DISCUSSION THREAD

**Discussion by alexbuga** [bangcorrupt/hacktribe#228] [Q&A] [answered]
I've successfully installed Hacktribe to a gray e2, but I'd like to install a factory e2s firmware 2.02 with factory samples (I actually like those bread & butter samples) and with the ability to add my on starting on slot 501.

Is this possible by just taking a stock e2s fw and just modifying headers without applying patch?

**Comment by alexbuga** [comment]
I remember years ago I bought 2 electribes and swapped the board between them and basically had a gray sampler. But I wouldn't do that today because it would be hard to sell the remaining one :))

**Comment by bangcorrupt** [comment, marked as answer]
It won't work like that, but I think there is a way.  To make it work properly we need to copy most of the flash contents, not just the firmware.  The next updates to Hacktribe Editor will allow reading and writing the flash memory via SysEx, with a (hopefully) user friendly interface.

**Reply by alexbuga** [reply]
Or actually, I guess what I'm trying to do is have the synth version with all the instrument PCM samples AND sampling capability :D

**Reply by bangcorrupt** [reply]
This is my goal with Hacktribe, full hybrid.
